jscomp/bsb/templates/basic/.gitignore

```diff
@@ -0,0 +1,24 @@
+*.exe
+*.obj
+*.out
+*.compile
+*.native
+*.byte
+*.cmo
+*.annot
+*.cmi
+*.cmx
+*.cmt
+*.cmti
+*.cma
+*.a
+*.cmxa
+*.obj
+*~
+*.annot
+*.cmj
+*.bak
+lib/bs
+*.mlast
+*.mliast
```

Comment:
Most of these seem unnecessary for a bsb-managed project. `lib/bs` will already exclude all bsb-produced intermediary artifacts. On the other hand, `.merlin` and `node_modules` are missing. My `.gitignore` is usually just:
```
node_modules
lib/bs
.merlin
```